NAMESPACE

```diff
@@ -73,5 +73,6 @@ importFrom("ppclust", "fcm", "pcm", "fpcm")
 importFrom("clue", "solve_LSAP")
 importFrom("ggpubr","ggarrange")
 importFrom("rtracklayer", "import")
-importFrom("BSgenome", "installed.genomes")
-
+importFrom("BSgenome", "installed.genomes", "getBSgenome")
+importFrom("kknn", "kknn")
+importFrom("stringr", "str_split")
```

Comment:
se nao ta usando essa funcao em nenhum lugar, se for usar precisa colocar esse pacote no imports do description, ou tira fora.